### PR TITLE
List jira tickets 0.0.1

### PR DESCRIPTION
**Pull request by a-mlc**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/list-jira-tickets/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+summary: |
+  Lists all JIRA tickets by comparing two branches.
+description: |
+  Lists all JIRA tickets by comparing two branches. It uses the JIRA ticket prefix to parse the commit messages. It will output either a list of ticket IDs or links if you add the URL path.
+website: https://bitbucket.org/silofit/bitrise-jira-tickets/src/master/
+source_code_url: https://bitbucket.org/silofit/bitrise-jira-tickets/src/master/
+support_url: https://bitbucket.org/silofit/bitrise-jira-tickets/src/master/
+published_at: 2021-08-17T15:09:12.991958-04:00
+source:
+  git: https://amarliac@bitbucket.org/silofit/bitrise-jira-tickets.git
```
Comment (godrei) [inline steps/list-jira-tickets/0.0.1/step.yml:11]: Why is the git URL different from the website and source code urls?

File: steps/list-jira-tickets/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+- opts:
+    description: |
+      JIRA ticket's prefix is necessary to extract the tickets when parsing the commit messages. It is following the pattern: [prefix]-[ticket number]. Example: "JTS" as a prefix will extract JIRA tickets like "JTS-1234"
+    is_expand: true
+    is_required: true
+    summary: null
+    title: JIRA ticket's prefix
+  prefix: null
+outputs:
+- JIRA_TICKET_IDS: null
```
Comment (godrei) [inline steps/list-jira-tickets/0.0.1/step.yml:64]: NOTE: having `_LIST` suffix for outputs with multiple values is the idiomatic way: https://devcenter.bitrise.io/contributors/create-your-own-step/#outputs-with-list-of-values

**Review by godrei — COMMENTED** [review]
@a-mlc thanks for the new Step, please review my comments.

File: steps/list-jira-tickets/0.0.1/step.yml (hunk 1 above)
Comment (a-mlc) [reply steps/list-jira-tickets/0.0.1/step.yml]: Addressed

File: steps/list-jira-tickets/0.0.1/step.yml (hunk 2 above)
Comment (a-mlc) [reply steps/list-jira-tickets/0.0.1/step.yml:64]: Addressed

**Pull request closed without merging**